jme3-core/src/main/resources/Common/MatDefs/Light/PBRLighting.frag

```diff
@@ -217,11 +217,11 @@ void main(){
         Roughness = 1.0 - glossiness;
         vec3 fZero = specularColor.xyz;
     #else
-        float specular = 0.5;
-        float nonMetalSpec = 0.08 * specular;
+        float nonMetalSpec = 0.08; //nonMetalSpec is always constant value of 0.8 in metallic workflow
```

Comment:
Comment: "0.08 in metallic workflow"! Classic example of how a comment describing existing code causes confusion ;)
Could probably reword it as that the specularity is always constant (or independent of (whatever it is usually)).

Reply:
Yeah, I think I should probably just remove that comment as well as the unnecessary nonMetalSpec variable declaration so I can condense the code. nonMetalSpec is just an internal constant variable that isn't used anywhere else anyways, so I'm not sure that it ever needed declared as a variable in the first place.